modules/shared/client/components/file-upload-drop.component.ts

```diff
@@ -0,0 +1,95 @@
+import {
+  Component,
+  EventEmitter,
+  forwardRef,
+  Input,
+  OnChanges,
+  Output,
+  SimpleChanges
+} from '@angular/core';
+import { FileUploader } from 'ng2-file-upload';
+import {
+  ControlValueAccessor,
+  NG_VALUE_ACCESSOR
+} from '@angular/forms';
+
+@Component({
+  selector: 'file-upload-drop',
+  templateUrl: './../views/file-upload-drop.view.html',
+  providers: [
+    {
+      provide: NG_VALUE_ACCESSOR,
+      useExisting: forwardRef(() => FileUploadDropComponent),
```

Comment:
setting up a token that says this component implements an interface so that it can be used directly in forms. ⚠️  this isn't working right now. The components work but they are not being used directly by the NgForm module. Eventually I'd like to get this working so validation can be done how it was before ⚠️